src/__tests__/downshift.get-item-props.js

```diff
@@ -178,6 +178,109 @@ test(`disabled item can't be selected by pressing enter`, () => {
   expect(input.value).toBe('c')
 })
 
+test(`getItemProps mapStateToProps function provides isSelected`, () => {
+  render(
+    <Downshift defaultSelectedItem={0}>
+      {({getItemProps}) => (
+        <div>
+          <span
+            {...getItemProps({item: 0}, ({isSelected}) => {
+              expect(isSelected).toBe(true)
```

Comment:
We should probably use [expect.assertions](https://jestjs.io/docs/en/expect#expectassertionsnumber) for any of these tests that put assertions within callbacks to make sure those assertions run.